src/Yubikey/Validate.php

```diff
@@ -269,7 +269,10 @@ public function check($otp, $multi = false)
 
         $clientId = $this->getClientId();
         if ($clientId === null) {
-            throw new \InvalidArgumentException('Client ID cannot be null');
+            if (!$this->getUseSecure()){
+                throw new \InvalidArgumentException('Client ID cannot be null');
+            }
+            $clientId = rand(1,9999);
```

Comment:
Is it proper to generate the random ```$clientId``` in that method?
I think it should use the [original approach](https://github.com/enygma/yubikey/blob/master/src/Yubikey/Validate.php#L279) is appropriate :+1: .